src/handlers/transfers/FxFulfilService.js

```diff
@@ -159,18 +202,30 @@ class FxFulfilService {
 
     // This is a duplicate message for a fxTransfer that is already in a finalized state
     // respond as if we received a GET /fxTransfers/{ID} from the client
-    if ([TransferState.COMMITTED, TransferState.ABORTED].includes(transferStateEnum)) {
-      this.params.message.value.content.payload = this.transform.toFulfil(transfer)
+    if (
+      [TransferState.COMMITTED, TransferState.ABORTED].includes(
```

Comment:
As for me, after such reformatting, readability got worse

Reply:
@geka-evk Could be the linter in the pre-commit hook. Will fix that.